lib/em-hiredis/pubsub_client.rb

```diff
@@ -181,33 +181,36 @@ def factory_connection
           @inactivity_response_timeout,
           @name
         )
-
         connection.on(:connected) {
           maybe_auth(connection).callback {
-
-            connection.on(:message, &method(:message_callbacks))
-            connection.on(:pmessage, &method(:pmessage_callbacks))
-
-            [ :message,
-              :pmessage,
-              :subscribe,
-              :unsubscribe,
-              :psubscribe,
-              :punsubscribe
-            ].each do |command|
-              connection.on(command) { |*args|
-                emit(command, *args)
+            connection.ping.timeout(@inactivity_response_timeout).callback {
```

Comment:
not sure if it's a good idea to use `@inactivity_response_timeout` or shall we just use smth like `timeout(2)`

Reply:
I think this is fine. I would maybe change the name to something like `@ping_response_timeout` though?

Reply:
I think `inactivity_response_timeout` is fine as it's consistent with the previous name.